standard/interfaces.md

```diff
@@ -436,138 +426,64 @@ A virtual method with implementation declared in an interface may be overridden
 
-```ANTLR
-interface_indexer_declaration
-    : attributes? 'new'? type 'this' '[' parameter_list ']'
-      '{' interface_accessors '}'
-    | attributes? 'new'? ref_kind type 'this' '[' parameter_list ']'
-      '{' ref_interface_accessor '}'
-    ;
-```
-
-Interface indexers are declared using *indexer_declaration*s ([§15.9](classes.md#159-indexers)).
+Interface indexers are declared using *indexer_declaration*s ([§15.9](classes.md#159-indexers)), with the following additional rules:
 
-*indexer_modifier* shall not include `override`.
+- *indexer_modifier* shall not include `override`.
+- An *indexer_declaration* that has a *block* as an *accessor_body* is an implementation ([§18.1](interfaces.md#181-general)), so it is *not* abstract.
+- All the parameter types of an interface indexer shall be input-safe ([§18.2.3.2](interfaces.md#18232-variance-safety)).
+- Any output or reference parameter types shall also be output-safe.
 
-An *indexer_declaration* that has a *block* as an *accessor_body* is a default implementation ([§18.1](interfaces.md#181-general)), so it is *not* abstract.
+  > *Note*: Output parameters are required to be input-safe due to common implementation restrictions. *end note*
 
-All the parameter types of an interface indexer shall be input-safe ([§18.2.3.2](interfaces.md#18232-variance-safety)). In addition, any output or reference parameter types shall also be output-safe.
-
-> *Note*: Output parameters are required to be input-safe due to common implementation restrictions. *end note*
-
-The type of an interface indexer shall be output-safe if there is a get accessor, and shall be input-safe if there is a set accessor.
+- The type of an interface indexer shall be output-safe if there is a get accessor, and shall be input-safe if there is a set accessor.
 
 ### §interface-operators Interface operators
 
 This clause augments the description of *operator_declaration* members in classes [§15.10](classes.md#1510-operators) for operators declared in interfaces.
 
-An *operator_declaration* in an interface is a default implementation ([§18.1](interfaces.md#181-general)).
+An *operator_declaration* in an interface is the implementation ([§18.1](interfaces.md#181-general)).
 
-It is a compile-time error for an interface to contain a conversion, equality, or inequality operator.
+It is a compile-time error for an interface to declare a conversion, equality, or inequality operator.
 
 ### §interface-static-constructors Interface static constructors
 
 This clause augments the description of static constructors in classes [§15.12](classes.md#1512-static-constructors) for static constructors declared in interfaces.
 
-A *static_constructor_declaration* that has a *block* as a *static_constructor_body* is a default implementation ([§18.1](interfaces.md#181-general)).
+A *static_constructor_declaration* that has a *block* as a *static_constructor_body* is the implementation ([§18.1](interfaces.md#181-general)).
```

Comment:
This leaves out expression-bodied static constructors, but I don't think we even need to make this conditional. It's also on the verge of "goes without saying" for me, since being virtual is impossible given what a static constructor _is_.
```suggestion
A *static_constructor_declaration* is the implementation ([§18.1](interfaces.md#181-general)).
```

Reply:
I did a bit more work here, for a few reasons:

1. static constructors can't be abstract or virtual (as in classes).
2. The static constructor defined in an interface may not be called if none of the members defined in that interface are called (meaning the interface definition is never the "most specific implementation".